packages/math/typesetter.lua

```diff
@@ -136,6 +136,19 @@ function ConvertMathML (_, content)
       return b.mtr(convertChildren(content))
    elseif content.command == "mtd" then
       return b.stackbox("H", convertChildren(content))
+   elseif content.command == "mtext" or content.command == "ms" then
+      if #content > 1 then
+         SU.error("Wrong number of children in " .. content.command .. ": " .. #content)
+      end
+      local text = content[1] or "" -- empty mtext is allowed, and found in examples...
+      if type(text) ~= "string" then
+         SU.error("ms command contains " .. text .. ", which is not text")
```

Comment:
Copied for other similar cases from above. But this crashes if text can't be serialized to string.

Reply:
Fixed.